Client/Frontend/Browser/URLBarView.swift

```diff
@@ -424,12 +424,10 @@ class URLBarView: UIView {
             // Clear any existing text, focus the field, then set the actual pasted text.
             // This avoids highlighting all of the text.
             locationTextField.text = ""
-            locationTextField.becomeFirstResponder()
```

Comment:
Wait maybe it's working now? I swear these tests start at random places in the test bundle. Still looking.